src/browser/subscription_manager.ts

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2017 OpenFin Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import ofEvents from './of_events';
+import route from '../common/route';
+import { Identity } from '../shapes';
+
+interface Subscription {
+    fn: () => void;
+    refCount: number;
+}
+
```

Comment:
It's only used in this file atm, we should move it out once/if it will also be required elsewhere